src/components/SimilarStories.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import { navigate } from '@reach/router'
+import {
+  EOS_THUMB_UP,
+  EOS_MESSAGE,
+  EOS_CALENDAR_TODAY_FILLED
+} from 'eos-icons-react'
+
+function Card(props) {
+  const { story } = props
+  return (
+    <div
+      className='card flex flex-column'
+      onClick={() => {
+        navigate(`/story/${story.id}`)
+      }}
+    >
+      <div>
+        <h3 className='link link-default subject'>{story.Title}</h3>

```

Comment:
let's change it for a <h4> since the HTML tree already has a parent h3, so for semantic reasons the children should be smaller (h4)

Reply:
it also works better UI wise, so win-win

Reply:
Done